gdk4/src/lib.rs

```diff
@@ -2,9 +2,6 @@
 
 #![cfg_attr(feature = "dox", feature(doc_cfg))]
 #![allow(clippy::type_complexity)]
-#![allow(clippy::derive_hash_xor_eq)]
-#![allow(clippy::non_send_fields_in_send_ty)]
-#![allow(clippy::too_many_arguments)]
```

Comment:
How did you find this?

Reply:
By running clippy multiple times :D

Reply:
You mean you removed them and checked if it failed?

Reply:
yes

Reply:
I guess that works, thanks :) I was hoping that you found an easier way